src/docs/operator-guides/migrating-from-outside-sharetribe/index.md

```diff
@@ -109,12 +215,202 @@ key like this:
+  - `:im.review/listing`
+
+Supported references
+
+- `#im.review/listing` -> `#im.listing`
+- `#im.review/author` -> `#im.user`
+- `#im.review/subject` -> `#im.user`
+
+Example syntax
+
+```
+[[:im.review/id]
+  #:im.review{:content "Exactly as advertised. Bummed this was a one time deal."
+              :rating 5
+              :type :review.type/ofProvider
+              :state :review.state/public
+              :createdAt #inst "2018-01-06T00:10:10Z"
+              :author #im/ref :user/jane
+              :subject #im/ref :user/john
+              :listing #im/ref :listing/rock-sauna}]
+```
+
+### Value types
+
+Intermediary format uses some notable value types:
+
+- [Location](#location)
+- [Money](#money)
+- [UUID](#uuid)
+- [Timestamp](#timestamp)
+
+#### Location
+
+Location is given as
+[LatLng type](https://docs.mapbox.com/android/maps/api/9.6.0/com/mapbox/mapboxsdk/geometry/LatLng.html)
+representing the latitude and longitude of the location. Both latitude
+and longitude are 32 bit floats. The `location` attribute is used in the
+FTW template to provide listing location coordinates through either
+Mapbox or Google maps.
+
+If your source data has addresses specified, and you would like to use
+them to determine locations, you can use e.g. the Mapbox
+[forwardGeoCode](https://github.com/sharetribe/ftw-daily/blob/01cdf40368380d64d90136efb37948c1da79ee0e/src/components/LocationAutocompleteInput/GeocoderMapbox.js#L106)
+API call in your data transformation setup. We recommend that you
+provide a valid location attribute if your marketplace uses maps in some
+way, whether or not you provide an address attribute in e.g. public
+data.
+
+```
+#:im.listing{...
+            :location #im/location [23.12 21.21]
+                      ...}
+```
+
+#### Money
+
+Money type is defined as [ amount, currency ]. _TODO What's the
+reasoning that price in Intermediary is given as decimals but Flex
+handles prices in minor units?_
+
+```
+#:im.listing{...
+            :price #im/money [12.12M "EUR"]
+            ...}
+```
+
+#### UUID
+
+A
+[Universally Unique identifiers (UUID)](https://en.wikipedia.org/wiki/Universally_unique_identifier)
+can be used to identify resources within the migration file using type
+`#uuid`. If your data already uses UUIDs, make sure their format matches
+[the canonical representation](https://en.wikipedia.org/wiki/Universally_unique_identifier#Format).
+
+```
+:im.image/id #uuid "58afd8e1-e336-4ca4-a1e7-ff1d91856a6c"
+```
+
+#### Timestamp
+
+Timestamp values are given with type `#inst`.
+
+```
+:createdAt #inst "2018-04-17T06:55:04.291-00:00"
+```
+
 ## Things to consider
 
 ### No updates are supported
 
-The import is run only once to the production environment and currently
-updates with multiple imports are not supported.
+The import is loaded only once to the production environment and
+currently updates with multiple imports are not supported.
```

Comment:
It could be worth mentioning that a migration can be run multiple times to test environment but as updates are not supported, this will result in duplicate data.